modules/apps/commerce/commerce-lang/src/main/resources/content/Language.properties

```
@@ -1554,6 +1554,7 @@ transaction-key=Transaction Key
 two-letter-iso-code=Two-Letter ISO Code
 type-here=Type Here
 type-your-note-here=Type your note here.
+unable-to-create-new-order-as-open-order-limit-has-been-reached=Unable to create New Order as open order limit has been reached.
```

@andrea-ale-sbarra I think this is more grammatically correct.

```suggestion
unable-to-create-a-new-order-as-the-open-order-limit-has-been-reached=Unable to create a new order as the open order limit has been reached.
```